doc/safety/safety_requirements.rst

```diff
@@ -0,0 +1,113 @@
+`Requirement repository
+<https://github.com/zephyrproject-rtos/reqmgmt>`__
+
+
+Guidelines
+**********
+Following are the guidelines for the requirements repository and what the safety committee
+expects when requirements are added to the repository
+
+Scope
+=====
+The scope of the requirements are the **KERNEL** functionalities.
+
+Consistency
+===========
+Consistency across all requirements. The language and choice of words shall be consistent.
+(See: `Syntax`_)
+
+Levels of requirements in the repository
+========================================
+* System Requirements:
+
+  System requirements describe the behaviour of the Zephyr RTOS (= the system here).
+  They describe the functionality of the Zephyr RTOS of a very high level point of view,
+  without going into details of the functionality itself.
+  The target of the system requirements is to get an overview of the currently implemented features
+  of the Zephyr RTOS.
+  In other words a person writing these requirements usually has some knowledge of the Zephyr RTOS
+  Project as the requirements are specific to an RTOS.
+
+* Software Requirements:
+
+  Software requirements refine the requirements of the system level at a more granular level so
+  that each requirement can be tested.
+  These requirements define the specific actions the feature shall be able to execute and the
+  behavior of the feature.
+
+Requirement UID (Unique identifier) Handling
+============================================
+For the requirements management the tool `strictDoc
+<https://strictdoc.readthedocs.io/en/stable/>`__
+is used, therefore the UID (Unique identifier) handling shall be handled by the tool and this can be achived by Following
+handling:
+
+#. Don't add a requirement UID filed for new requirements
+#. After finishing work on the new Requirements execute: **strictDoc manage auto-uid**
+#. Establish links between the requirements with the new distributed UIDs if needed
+
+After doing this the requirements are ready and a pull request can be created.
+The CI in the PR will check if the requirements UIDs are valid or if there are duplicates in it.
+If there are duplicates in the PR these need to be resolved by rebasing and re execute
+the steps above.
+
+Requirement Types
+=================
+* Functional
+* Non-Functional
+
+Requirement title convention
+============================
+Use short and terse requirement titles.
+
+Pull Request requirement repository
+===================================
+
+* Adhere to the :ref:`contribute_to_zephyr` of the Zephyr Project.
+
+  * As long as they are applicable to the requirements repository.
+
+* Avoid creating larger commits that contain both trivial and non-trivial changes.
+
+* Avoid moving and changing requirements in the same commit.
+
+Characteristics of a good requirement
+=====================================
+* Unambiguous
+* Verifiable (e.g. testable for functional requirements)
+* Clear (concise, terse, simple, precise)
```

Comment:
not a native speaker but I think "terse" is negatively connoted (i.e. means "abrupt")
you might want to switch to using "succinct" in your PR (there's 2 other occurences of the word)

Reply:
not a native speaker as well but the term terse is often used in requirements literature but will change it to concise